scripts/setup_integration/aws_integration_env.sh

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env bash
+set -o pipefail
+
+# aws_integration_env.sh
+#
+# SUMMARY
+#
+#   Builds and pulls down the Vector AWS Integration test environment
+
+set -x
+
+if [ $# -ne 1 ]
+then
+    echo "Usage: $0 {stop|start}" 1>&2; exit 1;
+    exit 1
+fi
+ACTION=$1
+
+#
+# Functions
+#
+
+start_podman () {
+  podman pod create --replace --name vector-test-integration-aws -p 4566:4566 -p 4571:4571 -p 6000:6000 -p 9088:80
+  podman run -d --pod=vector-test-integration-aws --name vector_ec2_metadata \
+  timberiodev/mock-ec2-metadata:latest
```

Comment:
Pretty much all of these scripts have something goofy going on with the indentation. It's not a huge issue, but it's a little annoying to look at for someone as obsessive about such details.